src/Common/src/Common.Utils/Steeltoe.Common.Utils.csproj

```
@@ -0,0 +1,14 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.0</TargetFrameworks>
+    <Description>Steeltoe common utility library</Description>
+    <PackageTags>NET Core;NET Framework</PackageTags>
```

```suggestion
    <PackageTags></PackageTags>
```
We dropped these tags from most packages in 3.1.0 as they're not that useful for finding packages and often Steeltoe.Common packages aren't expected to be used independently.